addons/website_forum/static/src/js/website_forum.js

```diff
@@ -264,7 +264,7 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
         ev.preventDefault();
         if (session.is_website_user) {
             this._displayAccessDeniedNotification(
-                Markup`<p>${sprintf(_t('Oh no! Please <a href="%s">sign in</a> to vote'), "/web/login")}</p>`
+                Markup`<p>${_t('Oh no! Please <a href="%s">sign in</a> to vote', "/web/login")}</p>`
```

Comment:
@flch-odoo FYI, this doesn't work: the term being translated will not be extracted within the interpolation sigils of template strings, it should be translated outside the template string. (same thing for the other string further down in this file). This probably should be fixed in stable

Reply:
@sdegueldre , thanks for the notice. Will do!